HackIllinois/UI/TableView/Cells/HIEventDetailLocationCell.swift

```diff
@@ -77,13 +77,14 @@ class HIEventDetailLocationCell: UITableViewCell {
 
     // MARK: - Themeable
     @objc func refreshForThemeChange() {
-        contentView.backgroundColor = HIApplication.Palette.current.contentBackground
-        switch HIApplication.Theme.current {
-        case .day:
-            blurEffectView.backgroundColor = nil
-        case .night:
-            blurEffectView.backgroundColor = HIApplication.Palette.current.contentBackground.withAlphaComponent(0.7)
-        }
+        contentView.backgroundColor = HIAppearance.current.contentBackground
+        // FIXME:
```

Comment:
you might want to put a note of some kind here for future reference or for other people to understand.

Reply:
fixed